modules/core/src/main/java/org/apache/ignite/internal/processors/continuous/GridContinuousProcessor.java

```diff
@@ -1998,6 +1998,9 @@ else if (loc != null) {
             // Removes routine at node started it when stopRoutine called from another node.
             unregisterHandler(routineId, loc.hnd, false);
         }
+
+        if (log.isInfoEnabled())
+            log.info("Continuous query was unregistered [routineId=" + routineId + ", loc=" + loc + ", rmt=" + remote + ']');
```

Comment:
Such logging can really spam a lot, why you chose this path instead of system view monitoring ?
How information about "routineId" will planned to be used ? I can`t see any "routineId" start message, only yours (appended in this PR)
Did you see what is finally printed ? 
`Continuous query was unregistered [routineId=d3df1e55-0fc3-4d6c-8809-18ca1b27e03c, loc=null, rmt=RemoteRoutineInfo [nodeId=d9a4452d-f93a-4c35-bb56-ecb6fd500000, hnd=CacheContinuousQueryHandler [returnValTrans=org.apache.ignite.internal.processors.cache.query.continuous.CacheContinuousQueryHandler$1@4261787, cacheName=default, rmtFilter=org.apache.ignite.internal.processors.cache.query.continuous.GridCacheContinuousQueryAbstractSelfTest$8@2d37d7c4, rmtFilterDep=null, internal=false, notifyExisting=false, oldValRequired=true, sync=false, ignoreExpired=true, taskHash=0, skipPrimaryCheck=false, locOnly=false, keepBinary=false, ackBuf=CacheContinuousQueryAcknowledgeBuffer [size=30, updateCntrs=HashMap {64=1, 1=1, 66=1, 4=1, 5=1, 70=1, 7=1, 8=1, 74=1, 12=1, 78=1, 15=1, 16=1, 80=1, 21=1, 24=1, 28=1, 31=1, 33=1, 34=1, 35=1, 40=1, 43=1, 46=1, 50=1, 54=1, 55=1, 56=1, 61=1, 63=1}], ackBufBackup=CacheContinuousQueryAcknowledgeBackupBuffer [size=30, updateCntrs=HashMap {64=1, 1=1, 66=1, 4=1, 5=1, 70=1, 7=1, 8=1, 74=1, 12=1, 78=1, 15=1, 16=1, 80=1, 21=1, 24=1, 28=1, 31=1, 33=1, 34=1, 35=1, 40=1, 43=1, 46=1, 50=1, 54=1, 55=1, 56=1, 61=1, 63=1}, topVers=HashSet [AffinityTopologyVersion [topVer=3, minorTopVer=1]]], cacheId=1544803905, initTopVer=null, nodeLeft=false, ignoreClsNotFound=false, nodeId=d9a4452d-f93a-4c35-bb56-ecb6fd500000, routineId=d3df1e55-0fc3-4d6c-8809-18ca1b27e03c], bufSize=1, interval=0, batch=org.apache.ignite.internal.processors.continuous.GridContinuousQueryBatch@2620d11c, lastSndTime=1730902141722, autoUnsubscribe=true, delayedRegister=false]]`
Huge excessive inner classes information + 
some sensitive data can be printed, is any who review that all 
`@GridToStringInclude(sensitive = true)` are on the correct place ?

Reply:
We can probably remove routed. It's CQ ( a long-running execution) so it should not spam much.

Reply:
I don`t understand your answer, but fill free to change a PR and i will check it once more, or feel free to connect me directly.